atlasdb-commons/build.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
     compile project(":commons-executors")
     compile 'javax.ws.rs:javax.ws.rs-api:2.0.1'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations'
+    compile 'net.jpountz.lz4:lz4:1.3.0'
```

Comment:
should probably put the version in https://github.com/palantir/atlasdb/blob/develop/versions.props and just rely on `compile 'net.jpountz.lz4:lz4'` here